table/queries/derived_satellite_scans.sql

```diff
@@ -73,6 +73,20 @@ CREATE TEMP FUNCTION AnswersSignature(answers ANY TYPE) AS (
   ), ",")
 );
 
+CREATE TEMP FUNCTION BadResolver(
+  resolver_non_zero_rcode_rate FLOAT64,
+  resolver_private_ip_rate FLOAT64,
+  resolver_zero_ip_rate FLOAT64,
+  resolver_connect_error_rate FLOAT64,
+  resolver_invalid_cert_rate FLOAT64
+) AS (
+  (resolver_non_zero_rcode_rate +
+   resolver_private_ip_rate +
+   resolver_zero_ip_rate +
+   resolver_connect_error_rate +
+   resolver_invalid_cert_rate) > 0.15
```

Comment:
It looks like the chart you used to derive this number is broken. Please revise.